### PR TITLE
docs: add readme with all the neccessary info about the project

### PR DESCRIPTION
**Pull request by AKushch1337**
The README will be modified further into the development cycle

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      - Cloud
+   - [Bouquet](https://github.com/GRizzi91/bouquet) framework
+   - Tests(UI Tests)
+      - jUnit4
+      - Compose UI testing tools
+      - Espresso
+
+## Build the project yourself
+
+- Clone the project
```
Comment (neliudochka) [inline README.md:36]: Add standard "git clone link" line

File: README.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+- Move into the project directory
```
Comment (neliudochka) [inline README.md:41]: add code to do so

File: README.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+- Build the project using Gradle Wrapper
+ ```gradle
+./gradlew build
+``` 
+- Or build it using Android Studio(that way you will run the app also!)
```
Comment (neliudochka) [inline README.md:50]: Extract information from the parenthesis to the Run section, or merge the Build and Run sections.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+## Run the project(easier way)
+
+- Download the APK from release page(can't right now)
+
+- Or run it using Android Studio
```
Comment (neliudochka) [inline README.md]: You repeat the same info in the 42 line

**Review by neliudochka — CHANGES_REQUESTED** [review]
If you have done fully UI, then add the Screenshots section

**Review by neliudochka — APPROVED** [review]
Now everything is good!

**Comment by AKushch1337** [comment]
Thank you!

**Pull request merged**